WoWPro/WoWPro_Frames.lua

```diff
@@ -1220,8 +1220,11 @@ function WoWPro:CreateNextGuideDialog()
 end
 
 
+local isResettingGuide = false
 function WoWPro.ResetCurrentGuide()
-    if not WoWProDB.char.currentguide then return end
+    if isResettingGuide then return end
```

Comment:
A simpler change would be:
```
function WoWPro.ResetCurrentGuide()
    if not WoWProDB.char.currentguide then return end
    if not WoWPro.GuideLoaded  then return end
    WoWPro.GuideLoaded = false
     …
 ```
 and use `WoWPro.GuideLoaded` as the guard.
 The use of the timer and flag will not be reliable.